### PR TITLE
Add speech recognition and dependencies targets

### PR DESCRIPTION
**Pull request by ar13pit**
resolves tue-robotics/tue_robocup#665

merge tue-robotics/speech_recognition#6 before this.

### CODE REVIEW

File: installer/targets/speech_recognition/install.bash [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (ar13pit) [reply installer/targets/speech_recognition/install.bash]: Yeah I was thinking about it. That's why didn't close this PR.

**Comment by ar13pit** [comment]
@MatthijsBurgh I think this is good to go now.

File: installer/targets/ros-speech_recognition/install.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+- type: target
```
Comment (MatthijsBurgh) [inline installer/targets/ros-speech_recognition/install.yaml]: These dependencies should be defined in the package.xml.

**Comment by ar13pit** [comment]
@MatthijsBurgh This is done.

**Pull request merged**